x-pack/solutions/observability/plugins/synthetics/public/apps/synthetics/components/monitor_add_edit/form/form_config.tsx

```diff
@@ -203,6 +203,17 @@ const TLS_OPTIONS = (readOnly: boolean): AdvancedFieldGroup => ({
   ],
 });
 
+const KIBANA_SPACES_OPTIONS = (readOnly: boolean): AdvancedFieldGroup => ({
+  title: i18n.translate('xpack.synthetics.monitorConfig.section.kibanaSpaces.title', {
+    defaultMessage: 'Kibana Spaces',
+  }),
+  description: i18n.translate('xpack.synthetics.monitorConfig.section.kibanaSpaces.description', {
+    defaultMessage:
+      'Select the Kibana spaces where this monitor should be available. If no space is selected, the monitor will be available in all spaces.',
+  }),
```

Comment:
Are these strings the same we have in `field_config.tsx`? Probably better not repeat them.